roles/connectivity_troubleshooter_peering/tasks/main.yml

```diff
@@ -9,7 +9,7 @@
       when: "'pcx-' not in connectivity_troubleshooter_validate__next_hop"
 
     - name: Gather information about peering connection
-      community.aws.ec2_vpc_peering_info:
+      amazon.aws.ec2_vpc_peering_info:
```

Comment:
This module was promoted to amazon.aws in [version 9.0.0](https://github.com/ansible-collections/amazon.aws/blob/main/changelogs/changelog.yaml#L3305).